marlowe-playground-client/src/Marlowe/Holes.purs

```diff
@@ -117,9 +119,9 @@ getMarloweConstructors BigIntegerType = Map.singleton "Integer" [ NewtypeArg ]
 
 getMarloweConstructors SlotType = Map.singleton "Integer" [ NewtypeArg ]
 
-getMarloweConstructors AccountIdType = Map.singleton "AccountId" [ DataArg "accNumber", DataArg "accHolder" ]
+getMarloweConstructors AccountIdType = Map.singleton "AccountId" [ DefaultNumber zero, DataArg "accHolder" ]
 
-getMarloweConstructors ChoiceIdType = Map.singleton "ChoiceId" [ DataArg "choiceNumber", DataArg "choiceOwner" ]
+getMarloweConstructors ChoiceIdType = Map.singleton "ChoiceId" [ DefaultString "choiceNumber", DataArg "choiceOwner" ]
 
 getMarloweConstructors ValueIdType = Map.singleton "ValueId" [ NewtypeArg ]
```

Comment:
I think `ValueId` has the `OverloadedString` too. It seems hole expansion already fails in alpha too, not sure

Reply:
The new tests cover this (plus I've fixed it)